idaes/gas_solid_contactors/unit_models/moving_bed.py

```diff
@@ -1,15 +1,15 @@
-#################################################################################
+###############################################################################
```

Comment:
A general observation (so don't go fixing it), but you generally shouldn't change this from the standard form. For one, there is a test that checks the header is present so you don't want to accidentally break that test.

Reply:
In general, three of the header lines run long and the linting check flags it. I'll leave them alone from now on.

Reply:
Yes, the linter does complain, but that means the header script should be updated instead.

Reply:
Oof, sorry for being caught in the middle of the two checkers squabbling with each other... I'll take a look (possibly together with @dangunter for the headers part)